### PR TITLE
add a package jsk_spot_joy_teleop

### PR DESCRIPTION
**Pull request by sktometometo**
add a package for controlling spot with the head-mounted joystick.

### CODE REVIEW

File: jsk_spot_robot/jsk_spot_joy_teleop/scripts/joy_topic_completion.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
+import rospy
+from sensor_msgs.msg import Joy
+ 
+class JoyTopicCompletion:
```
Comment (k-okada) [inline jsk_spot_robot/jsk_spot_joy_teleop/scripts/joy_topic_completion.py:5]: what is the role of this node? Does this try to move robot without pressing enable button? 
if so please add "README.md" page and show
- hardware setup
 - describe which harware (joystick) we're using, and how to install on the robot
 - add images, possibly animation gif, for experiments. something like https://ieeexplore.ieee.org/stamp/stamp.jsp?tp=&arnumber=676343

File: jsk_spot_robot/jsk_spot_joy_teleop/package.xml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>jsk_spot_joy_teleop</name>
+  <version>0.0.0</version>
```
[the remark this answers is not in the document]
Comment (sktometometo) [reply jsk_spot_robot/jsk_spot_joy_teleop/package.xml]: Changed the version to 1.1.0

File: jsk_spot_robot/jsk_spot_joy_teleop/scripts/joy_topic_completion.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+                buttons[self.enable_button] = 1
+                msg.buttons = tuple(buttons)
+
+        self.pub.publish(msg)
+
+    def __init__(self):
+
+	rospy.init_node('joy_topic_completion')
+	self.sub = rospy.Subscriber("~joy_input", Joy, self.callback)
+        self.pub = rospy.Publisher('~joy_output', Joy, queue_size=10)
```
[the remark this answers is not in the document]
Comment (sktometometo) [reply jsk_spot_robot/jsk_spot_joy_teleop/scripts/joy_topic_completion.py:24]: Fixed the format of joy_topic_completion.py

File: jsk_spot_robot/jsk_spot_joy_teleop/scripts/joy_topic_completion.py (hunk 1 above)
Comment (sktometometo) [reply jsk_spot_robot/jsk_spot_joy_teleop/scripts/joy_topic_completion.py:7]: added README.md which contains
- Hardware description
- Demo gif

**Pull request merged**